index.html

```diff
@@ -1,24 +1,137 @@
-<doctype html>
+<!doctype html>
 <html>
 
 <head>  
 <title> </title>
 <style type="text/css">
 	
 </style>
-
+<link type="text/css" rel="stylesheet" href="stylesheet.css" >
 </head>
 
 <body>
+<div>
+  <div id="navbar">
+	<div id="menu">
+		<button id="menu_button"><span><img src="images/menu-icon.png" ></span></button>
+	</div>
+	<div id="logo">
+		<a href="#"><img src="images/youtube-icon.png" height="25px" width="80px" ></a>
```

Comment:
Please, always use `alt` attribute for `<img>` element. :bulb: 

This is a very important accessibility attribute.
`Examples of scenarios where users benefit from text alternatives for images` are listed below and could also be found [here](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)

- They have a very slow connection and are browsing with images disabled.
- They have a vision impairment and use text to speech software.
- They have a cognitive impairment and use text to speech software.
- They are using a text-only browser.
- They are listening to the page being read out by a voice Web browser.
- They have images disabled to save on download costs.
- They have problems loading images or the source of an image is wrong.


##### SOURCE: [W3C Recommendation (Embedded Content)](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)

Reply:
Thanks for your review. This has been improved on.